mappings/net/minecraft/entity/passive/CamelEntity.mapping

```diff
@@ -0,0 +1,43 @@
+CLASS net/minecraft/unmapped/C_lzclwwts net/minecraft/entity/passive/CamelEntity
+	FIELD f_chodqqna DASH_VERTICAL_MOMENTUM F
+	FIELD f_gxjuclwy RUNNING_SPEED_BONUS F
+	FIELD f_jnrgxjnd DIMENSIONS Lnet/minecraft/unmapped/C_sszpscpo;
+	FIELD f_jwcjrvxw DASH_HORIZONTAL_MOMENTUM F
+	FIELD f_jyjjptyj standUpState Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_katvxrqz STANDUP_DURATION_TICKS I
```

Comment:
"Standup" in this context is not a word, it would be "Stand up"

Reply:
```suggestion
	FIELD f_katvxrqz STAND_UP_DURATION_TICKS I
```